streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/ProcessorNodeMetricsTest.java

```diff
@@ -114,6 +116,63 @@ public void shouldGetProcessAtSourceSensor() {
         verifySensor(() -> ProcessorNodeMetrics.processAtSourceSensor(THREAD_ID, TASK_ID, PROCESSOR_NODE_ID, streamsMetrics));
     }
 
+    @Test
+    public void shouldGetRecordsAndBytesConsumedSensor() {
+        final String recordsMetricNamePrefix = "records-consumed";
+        final String bytesMetricNamePrefix = "bytes-consumed";
+        final String descriptionOfRecordsTotal = "The total number of records consumed from this topic";
+        final String descriptionOfBytesTotal = "The total number of bytes consumed from this topic";
+        when(streamsMetrics.nodeLevelSensor(THREAD_ID, TASK_ID, PROCESSOR_NODE_ID, recordsMetricNamePrefix, RecordingLevel.INFO))
+            .thenReturn(expectedSensor);
+        when(streamsMetrics.nodeLevelSensor(THREAD_ID, TASK_ID, PROCESSOR_NODE_ID, bytesMetricNamePrefix, RecordingLevel.INFO))
+            .thenReturn(expectedSensor);
+        when(streamsMetrics.nodeLevelTagMap(THREAD_ID, TASK_ID, PROCESSOR_NODE_ID)).thenReturn(tagMap);
+
+        final Map<String, String> consumedTagMap = new HashMap<>(tagMap);
+        consumedTagMap.put("topic", TOPIC_NAME);
+        StreamsMetricsImpl.addTotalCountAndSumMetricsToSensor(
```

Comment:
Ah, thanks for the explanation. I updated the test as suggested (now in `TopicMetricsTest`)

Reply:
Oh man I had no idea you had to close static mocks...not blaming you for this at all @cadonna but in the future, assume I know nothing about Mockito/PowerMock and put everything out on the table! lol 🙃

Reply:
Oh no! I just prototyped that code since it didn't make sense how the code was before with Mockito. I did also not know about closing static mocks in Mockito. I am really sorry!